### DISCUSSION TITLE
show line numbers and set other octo filetype specific variables

### DISCUSSION THREAD

**Discussion by divramod** [pwntester/octo.nvim#316] [Q&A]
hey,

i am using the vim line numbers to navigate.

is it somehow possible to show line numbers in octo buffers?

i tried to use a file ```octo.vim``` in ftplugin but it seems to have no effect.

can anyone give me a hint into the right direction?

thx in advance

**Comment by pwntester** [comment]
They are disabled [here](https://github.com/pwntester/octo.nvim/blob/1f6a770a3b2a463cff474df51dc037ae7a6c2920/lua/octo/model/octo-buffer.lua#L242) but you should be able to add an autocmd that re-enables them on `BufEnter` autocmd

**Reply by divramod** [reply]
ah, thx for the reply.

i tried ```autocmd BufEnter octo* set number``` but with no effect.

do you think it's possible to create a config option for this?

**Reply by artur-sannikov** [reply]
I don't think it's an option here. When running

```
:set number
```

I visually see that some space is created on the left, but the numbers themselves do not appear.

**Comment by TyceHerrman** [comment]
I think the workaround suggested above is outdated now after #555.

Octo owns `statuscolumn` for comment/review markers. Once a custom `statuscolumn` is set, neovim line numbers do not render unless that statuscolumn includes them, and Octo’s statuscolumn does not compose with line numbers.

maybe there could be an option like

```lua
ui = {
  use_signcolumn = true,
  use_statuscolumn = true,
  statuscolumn_line_numbers = true,
}
```
